docs/pages/transition-patterns.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Transition patterns
+layout: variation
+section: patterns
+status: Released
+description: Transition patterns are animations that happen when a user
+  interacts with an element on the page. They are CSS transition styles that are
+  controlled via JavaScript.
+variation_groups:
+  - variations:
+      - variation_name: Move transition
+        variation_description: A transition that moves from one position to another.
+        variation_code_snippet: <div class="u-move-transition example-box">Click me!</div>
```

Comment:
Is there a way to keep the example JS next to the example HTML? What if we changed the code snippet to:

```html
<div class="u-move-transition example-box">Click me!</div>
<script>
    const moveTransitionExample = document.querySelector( '.example-box.u-move-transition' );
    const moveTransition = new MoveTransition( moveTransitionExample ).init();
    moveTransitionExample.addEventListener( 'click', () => {
        moveTransition.moveRight();
    } );
</script>
```

And then removed the JS from `main.js`? IMHO it would make the implementation details clearer.

Reply:
Hmmmmmmmmmm yea that makes sense.